dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/newnonfinancials/FinancialShare.kt

```diff
@@ -0,0 +1,12 @@
+package org.dataland.datalandbackend.model.eutaxonomy.nonfinancials
+
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * This class describes a the relative and absolute share on a different financial asset
```

Comment:
Typo in the desription. One "a" too much.

Reply:
fixed

Reply:
ok